sdk/core/azure-core/azure/core/pipeline/policies/__init__.py

```diff
@@ -43,9 +48,10 @@
 __all__ = [
     'HTTPPolicy',
     'SansIOHTTPPolicy',
-    'BearerTokenCredentialPolicy',
     'AzureKeyCredentialPolicy',
     'AzureSasCredentialPolicy',
+    'BearerTokenCredentialPolicy',
+    'MultitenantTokenCredentialPolicy',
```

Comment:
Do we need an async one?

Reply:
We will! That is on its way, along with unit tests 🙂

Reply:
And you will have a separate PR for it or will merge it into this PR?

Reply:
That will be part of this PR